api/src/main/java/org/openmrs/liquibase/ChangeLogVersionFinder.java

```diff
@@ -28,15 +29,13 @@
  */
 public class ChangeLogVersionFinder {
 	
-	static final String BASE_FOLDER_NAME = "org" + File.separator + "openmrs" + File.separator + "liquibase";
+	static final String BASE_FOLDER_NAME = Paths.get("org",  "openmrs", "liquibase").toString();
 	
-	static final String CORE_DATA_FOLDER_NAME = BASE_FOLDER_NAME + File.separator + "snapshots" + File.separator
-	        + "core-data";
+	static final String CORE_DATA_FOLDER_NAME = Paths.get(BASE_FOLDER_NAME, "snapshots", "core-data").toString();
 	
-	static final String SCHEMA_ONLY_FOLDER_NAME = BASE_FOLDER_NAME + File.separator + "snapshots" + File.separator
-	        + "schema-only";
+	static final String SCHEMA_ONLY_FOLDER_NAME = Paths.get(BASE_FOLDER_NAME,"snapshots", "schema-only").toString();
```

Comment:
Was spacing necessary here  `BASE_FOLDER_NAME,"snapshots"`

Reply:
Spacing is present in other instances.

https://github.com/openmrs/openmrs-core/blob/d5d65c823890ce533d70d5e308eaccf59d3478a4/liquibase/src/main/java/org/openmrs/liquibase/Main.java#L19-L29

Reply:
okk